components/esp-tls/esp_tls.c

```diff
@@ -566,6 +566,54 @@ mbedtls_x509_crt *esp_tls_get_global_ca_store(void)
 
 #endif /* CONFIG_ESP_TLS_USING_MBEDTLS */
 #ifdef CONFIG_ESP_TLS_SERVER
+
+#if defined(CONFIG_ESP_TLS_USING_MBEDTLS) && defined(CONFIG_ESP_TLS_SERVER_SESSION_TICKETS)
+
+int esp_tls_session_ticket_ctx_init(esp_tls_session_ticket_ctx_t * ctx) {
+
+    mbedtls_ctr_drbg_init(&ctx->ctr_drbg);
```

Comment:
I think this code can be moved to `esp_tls_mbedtls.c` , maybe add a similar function in `esp_tls_mbedtls.h` and include it here.

Reply:
I have moved the function definition and declaration to esp_tls_mbedtls.*